samples/display/st7789v/boards/nrf52_pca10040.conf

```diff
@@ -0,0 +1,6 @@
+CONFIG_SPI=y
+CONFIG_GPIO=y
+
+CONFIG_ST7789V=y
+CONFIG_ST7789V_LCD_TL019FQV01=y
+CONFIG_ST7789V_RGB565=y
```

Comment:
Nothing in this file appears to be specific to the nrf board. Is there a reason you didn't just put it all in the `proj.conf` file?

Reply:
move these to proj.conf